tests/check_consistent.py

```diff
@@ -162,46 +130,8 @@ def _find_stdlib_modules() -> set[str]:
 
 def check_metadata() -> None:
```

Comment:
At this point, this feels redundant since other tests will fail anyway.
Unless you wanna use it as a smoke test.

Reply:
I get what you mean, but the other tests will assert the correctness of the METADATA.toml files "by accident". I feel like it's useful to have a test that asserts the correctness of the METADATA.toml files "on purpose". That way we still have this guaranteed to be tested in CI, even if some other tests no longer need to get stuff from METADATA.toml files in the future, for whatever reason.

Reply:
Yeah that's a smoke test (an easy test that says, if this fails, everything else will fail anyway, so it's easy to find the source of the error). Just wanted to check if it was on purpose :)